var/spack/repos/builtin/packages/py-easybuild-framework/package.py

```diff
@@ -12,11 +12,14 @@ class PyEasybuildFramework(PythonPackage):
     """
 
     homepage = 'http://hpcugent.github.io/easybuild/'
-    url      = 'https://pypi.io/packages/source/e/easybuild-framework/easybuild-framework-3.1.2.tar.gz'
+    url      = 'https://pypi.io/packages/source/e/easybuild-framework/easybuild-framework-4.0.0.tar.gz'
+    maintainers = ['boegel']
 
+    version('4.0.0', sha256='f5c40345cc8b9b5750f53263ade6c9c3a8cd3dfab488d58f76ac61a8ca7c5a77')
     version('3.1.2', '283bc5f6bdcb90016b32986d52fd04a8')
 
-    depends_on('python@2.6:2.8', type='run')
-    depends_on('py-setuptools', type=('build', 'run'))
+    depends_on('python@2.6:2.8', when='@:3', type=('build', 'run'))
+    depends_on('python@2.6:2.8,3.5:', when='@4:', type=('build', 'run'))
+    depends_on('py-setuptools', when='@:3', type=('build', 'run'))
     depends_on('py-vsc-base@2.5.4:', when='@2.9:', type='run')
```

Comment:
`vsc-base` is no longer required for EasyBuild 4.0.0 and newer (see https://easybuild.readthedocs.io/en/develop/EasyBuild4-overview-of-changes.html#no-more-required-python-packages)